test/unit/cuda/test-reduce-loc.cpp

```diff
@@ -167,7 +167,7 @@ CUDA_TYPED_TEST_P(ReduceCUDA, generic)
       reduce::detail::ValueLoc<double> randval(droll, index);
       applier::updatedvalue(dvalue, randval, dcurrent);
 
-      forall<cuda_exec<block_size>>(0, TEST_VEC_LEN, [=] __device__(int i) {
+      forall<cuda_exec<block_size>>(RAJA::RangeSegment(0, TEST_VEC_LEN), [=] __device__(int i) {
```

Comment:
Is the `RangeSegment` alias going to be deprecated for `TypedRangeSegment` any time soon?

Reply:
I don't see it on the TODO list. @rhornung67, will we be keeping the RangeSegments?

Reply:
I think we should leave the type aliases for RangeSegment and RangeStrideSegment in place for a while at least. We should discuss as a group whether to deprecate and, if so, when.

Reply:
Sounds good to me.